src/main/java/encryptor.java

```diff
@@ -18,29 +17,24 @@ public static void main(String[] args) {
             System.out.println("menu: \n 1: encryption \n 2: decryption");
             int input = scanner.nextInt();
             scanner.nextLine();
-            IEncryptionAlgorithm encryptionAlgorithm =new RepeatEncryption(new XorEncryption(),5);
+            IEncryptionAlgorithm encryptionAlgorithm =new RepeatEncryption(XorEncryption.createXorEncryption(),5);
             FileEncryptor fileEncryptor = new FileEncryptor(encryptionAlgorithm);
             if (input == 1) {
                 //get the file-path from the user
                 System.out.println("Enter the path of your source file:");
                 String stringPath = scanner.nextLine();
                 stringPath = stringPath.replaceAll("\"", "");
-
+                try {
                 Path path = Paths.get(stringPath);
                 //calculate the path
-                Path filesPath = FileStream.getOutputFilesPath(path);
+                Path filesPath = path.getParent();
                 Path outputFilepath = FileStream.getFileName(path, "_encrypted");
 
-                try {
                     fileEncryptor.encryptFile(path,outputFilepath,filesPath);
+                    System.out.println("the locations of the key file and the encrypted file :" + filesPath + "\n");
```

Comment:
Use capital letter at the begging of the log.
The println method will add new line automatically at the end of the log